eventloop/eventloop.go

```diff
@@ -250,6 +250,11 @@ func (loop *EventLoop) addAuxJob(fn func()) {
 }
 
 func (loop *EventLoop) addTimeout(f func(), timeout time.Duration) *Timer {
+	// If no timeout is provided set timeout to 1ms.  Follows node documentation.
+	// https://nodejs.org/api/timers.html#timers_settimeout_callback_delay_args
```

Comment:
Interestingly the browser specs are at odds with nodejs, because according to them 'If this parameter is omitted, a value of 0 is used, meaning execute "immediately", or more accurately, the next event cycle' (https://developer.mozilla.org/en-US/docs/Web/API/setTimeout).

I would probably leave this function as is and only change addInterval() as the most important thing is to fix the panic.